hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/DirectoryDeletingService.java

```diff
@@ -158,7 +215,8 @@ public BackgroundTaskResult call() {
           LOG.debug("Running DirectoryDeletingService");
         }
         isRunningOnAOS.set(true);
-        getRunCount().incrementAndGet();
+        // rnCnt will be same for each thread to maintain same CallId.
+        long rnCnt = getRunCount().incrementAndGet();
```

Comment:
Comment looks wrong. rnCnt is different for each request/thread and so CallId will be different which is correct.

Reply:
Removed comment as discussed.